.github/workflows/master_build_pr.yml

```diff
@@ -1051,6 +1051,20 @@ jobs:
             cd ..
             set +e
           }
+          vssTarBuild() {
+            set -e
+            ARTIFACT="$1"
+            EXTENSION="$2"
+            CLOUD=$(echo "$ARTIFACT" | awk -F'[-]' '{print $3}')
+            DIR="build_$CLOUD"
+            mkdir $DIR
+            cp -r  ../tessell-vss/tessell-vss-requestor/TessellVssRequestor.exe ../tessell-vss/tessell-vss-provider/$CLOUD/install-provider $DIR
```

Comment:
Given that this file (`TessellVssRequestor.exe`) is NOT generated but added /modified by developer, should we be creating artifact Only if there is a change in it.

Reply:
Lets keep it as it is. Creating tar only on change of .exe will need more changes on release-management side.